apis/v1alpha1/route_types.go

```diff
@@ -25,12 +25,63 @@ type GatewayObjectReference struct {
 	// +optional
 	Namespace string `json:"namespace,omitempty"`
```

Comment:
Should min/max length be established for `Name` and `Namespace` fields?

Reply:
Updated, thanks!